.devcontainer/compose.yaml

```diff
@@ -21,12 +21,13 @@ services:
       ES_HOST: es
       ES_PORT: '9200'
       LIBRE_TRANSLATE_ENDPOINT: http://libretranslate:5000
+      LOCAL_DOMAIN: ${LOCAL_DOMAIN:-127.0.0.1:3000}
```

Comment:
The default `LOCAL_DOMAIN` is `localhost:3000`, it's not a big difference but looks a bit less wild in the UI

Reply:
fixed in 8ed6011